models/scripts/SentimentAnalysis.py

```diff
@@ -0,0 +1,59 @@
+from tabpy_tools.client import Client
+from textblob import TextBlob
+from nltk.sentiment import SentimentIntensityAnalyzer
+import sys
+from pathlib import Path
+sys.path.append(str(Path(__file__).resolve().parent.parent.parent / 'models'))
+from utils import setup_utils
+
+
+def SentimentAnalysis(_arg1, library='nltk'):
+    '''
+    Sentiment Analysis is a procedure that assigns a score from -1 to 1
+    for a piece of text with -1 being negative and 1 being positive. For
+    more information on the function and how to use it please refer to
+    tabpy-tools.md
+    '''
+    assert (type(_arg1[0]) is str)
+
+    library = library.lower()
+    supportedLibraries = {'nltk', 'textblob'}
+
+    assert (library in supportedLibraries)
```

Comment:
Just so you are aware assertion checks get stripped out if Python is run with `-OO` optimization, which some users may opt for. Typically better and more descriptive to raise some other type of error (here maybe a ValueError for the library)

Reply:
I removed all of the asserts and raised errors instead